control/lti.py

```diff
@@ -159,6 +159,7 @@ def damp(self):
         poles = self.pole()
 
         if isdtime(self, strict=True):
+            poles = poles.astype(complex)
             splane_poles = np.log(poles)/self.dt
```

Comment:
This also makes the third return parameter of `.damp()` complex, even if it has no imaginary component. Let's keep it real.

```suggestion
            splane_poles = np.log(poles.astype(complex))/self.dt
```